imagetest/test_suites/image_validation/image_validation_test.go

```diff
@@ -64,14 +64,21 @@ func TestFQDN(t *testing.T) {
 	}
 
 	// This command is not safe on multi-NIC VMs. See HOSTNAME(1), section 'THE FQDN'.
-	cmd := exec.Command("/bin/hostname", "-A")
+	cmd := exec.Command("/bin/hostname", "-a")
```

Comment:
-a does a totally different thing. it must be -A

Reply:
You are correct. hostname -A is correct.